package.json

```diff
@@ -12,19 +12,19 @@
   },
   "private": true,
   "dependencies": {
-    "@angular/animations": "^4.3.0",
```

Comment:
We have a `yarn.lock` file, but it is outdated now. We should probably remove it and optionally add the `package-lock.json`.

Reply:
Will update here as well.

Reply:
This is fixed and `package-lock.json` added.